app/models/user.rb

```diff
@@ -1,4 +1,7 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ Required ] I think it would be better if you ensure that before a user can log in into the application a user can `confirm` their email before they can log in into the application. To make this possible you just need to add the `:confirmable` module in the devise gem.

## Example code
```ruby
  devise :database_authenticatable, :registerable,
         :recoverable, :rememberable, :validatable, :confirmable
```